fem/fespace.hpp

```diff
@@ -647,6 +647,8 @@ class FiniteElementSpace
 
    void GetEdgeInteriorDofs(int i, Array<int> &dofs) const;
 
+   void GetVDofs(int vd, Array<int> &dofs, int ndofs = -1) const;
```

Comment:
Docstring would be good here (e.g. explaining what `ndofs = -1` signifies)

Reply:
Agreed.  Added in the latest commit.